Dockerfile

```
@@ -1,8 +1,10 @@
 # pull container from https://hub.docker.com/_/ros, choosing the melodic-robot-bionic version
 FROM ros:melodic-robot-bionic
 
-# install dependencies for the F1Tenth simulator and wget
-RUN sudo apt-get update
+# install dependencies for the F1Tenth simulator and wget (and now pip)
+RUN apt-get update && apt-get install -y python3-pip
```

```suggestion
RUN sudo apt-get update
```
We can actually keep this line the same and then add the `python3-pip` to the installs on the next line. I'll suggest that below